article/sections/_4_graph_construction.tex

```diff
@@ -6,64 +6,67 @@ \section{(Initial) Causal Graph Construction}
 \begin{enumerate}
    \item adopt a population perspective,
    \item brainstorm all variables that we think affect the system that generates our observations,
-   \item remove any variables that would cause bias in our causal inferences,
-   \item connect all variables in our graph according to our a-priori beliefs about causal relations amongst these variables,
+   \item remove any variables that could cause bias in our causal inferences,
+   \item connect all variables in our graph according to our a-priori beliefs about causal relations amongst them
    \item consider how the graph structure may differ between individuals and subgroups within the population.
 \end{enumerate}
-The following paragraphs will describe these steps in detail.
+The following paragraphs describe these steps in detail.
 
 To begin, we adopt the position of a researcher concerned about population level relationships.
 This means we will think through what is a likely generative model for all individuals.
-Later on, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
+Later in this section, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
 
-Now, we add our first variables to our graph, the outcome variable(s) of interest.
-Then, we note all the variables that we believes to influence the variable(s) of interest.
-We refer to these variables to as our initial explanatory variables.
+Now, we add our first variable(s) to our graph, the outcome variable(s) of interest in our problem.
+Then, we list all the variables we believe to influence the outcome(s).
+We refer to these variables as our initial explanatory variables.
 
-Next, we iterate through the initial explanatory variables.
+Next, we iterate through these initial explanatory variables.
 For each current explanatory variable in the iteration, we think of variables that may modify the effect of the current explanatory variable on the outcome(s) of interest.
 We refer to these variables as effect modifiers.
```

Comment:
I think an example of an effect modifier would be extremely useful. As it stands they can be easily confused with confounders and/or mediators, which _modify_ the effect too.

Reply:
Added!